src/show.jl

```diff
@@ -21,3 +21,42 @@ Base.show(io::IO, A::DiskGeoArray) = begin
     end
     print(io, "\n  From file: $(filename(A))")
 end
+
+function Base.show(io::IO, stack::AbstractGeoStack)
+    printstyled(io, "$(Base.typename(typeof(stack)))", color=:blue)
+    print(io, " with $(length(keys(stack))) field(s)")
+    applicable(filename, stack) && print(io, ": $(filename(stack))")
+    print(io, '\n')
+
+    for var in keys(stack)
+        printstyled(io, " $var", color=:green)
+
+        field_dims = dims(stack, var)
+        if length(field_dims) > 0
+            print(io, " with dimension(s) ")
+            for (d, dim) in enumerate(field_dims)
+                printstyled(io, "$(GeoData.name(dim))($(length(dim)))", color=:red)
```

Comment:
Lets change the `Dim(length)` display, it looks like a method call, and it's not clear that number means length...

Maybe:
```
Dim [length: 45]
```
Or something like that that looks better. Then you can also have the bracket not be red

Reply:
Done.